ledger/acctonline.go

```diff
@@ -725,10 +725,10 @@ func (ao *onlineAccounts) lookupOnlineAccountData(rnd basics.Round, addr basics.
 	}
 }
 
-// onlineTop returns the top n online accounts, sorted by their normalized
+// OnlineTop returns the top n online accounts, sorted by their normalized
 // balance and address, whose voting keys are valid in voteRnd.  See the
 // normalization description in AccountData.NormalizedOnlineBalance().
-func (ao *onlineAccounts) onlineTop(rnd basics.Round, voteRnd basics.Round, n uint64) ([]*ledgercore.OnlineAccount, error) {
+func (ao *onlineAccounts) OnlineTop(rnd basics.Round, voteRnd basics.Round, n uint64) ([]*ledgercore.OnlineAccount, error) {
```

Comment:
we were just talking about `voteRnd` with @zeldovich and @algoidan and how it knocks out accounts whose keys have expired between `rnd` and `voteRnd` (which is rnd+StateProofVotersLookback+StateProofInterval). The explanation was that this used to share participation keys with agreement, which used its own ephemeral key deletion schedule.

So now that state proofs manage their own ephemeral keys — the round used to pull keys used by the signer could match the lookback round? It looks like the signer is currently calling AccountManager.StateProofKeys to get keys for rnd+StateProofVotersLookback+StateProofInterval?